### PR TITLE
Interface implementation with working client routing

### PR DESCRIPTION
**Pull request by Gordienko-RU**
User interface implementation(only structure, without pretty decoration), and client-routing realization(there are two main navigation-links and content-switching depends on current url).

Interface consists of three main blocks:
-breadcrumbs;
-navigation;
-work-area.

Work-area will contain list and profile for both - users and groups, and some home content by default. 
Because all pages will contain navigation and breadcrumbs - they placing outside work-area block, so we don't need to call it inside each page.  

Looks like this https://gyazo.com/86cbc60cafda31c79f9ed8d8ec668dae

Styles were implemented via js(**jss** lib)

### CODE REVIEW

File: src/components/Breadcrumbs/Breadcrumbs.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import classes from './breadcrumbs-style';
+
+const {breadcrumbs} = classes;
```
Comment (dimapaloskin) [inline src/components/Breadcrumbs/Breadcrumbs.jsx]:
```js
{ breadcrumbs }
```
with spaces, dude :)

File: src/components/Breadcrumbs/breadcrumbs-style.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import jss from '../../styles/customized-jss';
```
Comment (dimapaloskin) [inline src/components/Breadcrumbs/breadcrumbs-style.js]: what the fuck?

File: src/components/Navigation/Navigation.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { Link } from 'react-router-dom';
+import React from 'react';
+import navigationContent from './navigation-content';
+import classes from './navigation-style';
+
+const {navigation, navLink} = classes;
```
Comment (dimapaloskin) [inline src/components/Navigation/Navigation.jsx]: Let's put spaces in any destruction. In doesn't touch only this line. it's related with whole project.

File: src/components/AdminPanel.jsx [hunk 4]
```diff
@@ -1,9 +1,15 @@
 import React from 'react';
+import Breadcrumbs from './Breadcrumbs/Breadcrumbs.jsx';
```
Comment (dimapaloskin) [inline src/components/AdminPanel.jsx]: Please, let's avoid duplicate names of the entity. If some entity has no any dependency - put it as a single file.

So:

`components/Breadcrumbs/Breadcrumbs.jsx`
goes to
`components/Breadcrumbs.jsx`

You will fill when folder will be needed \m/

Comment (Gordienko-RU) [reply src/components/AdminPanel.jsx]: Fixed it according airbnb style-guide.

File: src/components/Navigation/Navigation.jsx (hunk 3 above)
Comment (Gordienko-RU) [reply src/components/Navigation/Navigation.jsx]: Got it.

File: src/components/Breadcrumbs/breadcrumbs-style.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+const styles = {
```
[the remark this answers is not in the document]
Comment (dimapaloskin) [reply src/components/Breadcrumbs/breadcrumbs-style.js]: It's related not only with this file

File: src/components/WorkArea/index.jsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
+import { Switch, Route } from 'react-router-dom';
+import React from 'react';
+import injectSheet from 'react-jss';
+import styles from './work-area-style';
+import UserList from './work-area-content/UserList';
```
Comment (dimapaloskin) [inline src/components/WorkArea/index.jsx]: Just `content`.

In content WorkArea it clear

**Pull request merged**